wasm.md

```diff
@@ -204,14 +204,15 @@ Function `#unsigned` is called on integers to allow programs to use negative num
 
 ### Text Format Conventions
 
-The text format allows the use of symbolic `identifiers` in place of `indices`.
-To resolve these `identifiers` into concrete `indices`, some grammar production are indexed by an identifier context `I` as a synthesized attribute that records the declared identifiers in each index space. We call this operation `ICov`.
+The text format allows the use of symbolic `identifiers` in place of `addrs`.
```

Comment:
Hmm.. What does it mean? I think the symbolic `identifiers` are representing the `index` in the module instance. `addr` should be in the store and has nothing to do with the `identifier`

Reply:
Ah, good catch. This seems like the result of a global search&replace.

Reply:
It seemed the documentation was slightly out of date here, so I updated it and extended it a bit.